### PR TITLE
Escape user supplied data within the page template

### LINKED ISSUE #9: DOM based XSS via params
It seems the exception page parses params dynamically in the DOM, it's really simple to add any param to the query and use the basic XXS token like `<script>alert(1)</script>` to execute the attack.

Screenshot at 15-55-45

### PR DESCRIPTION
**Pull request by Sija**
Closes #9

### CODE REVIEW

File: src/exception_page/exception_page.ecr [hunk 1]
```diff
@@ -18,7 +18,7 @@ monospace_font = "menlo, consolas, monospace"
 <html>
 <head>
     <%
-        details = @message.split('\n')
+        details = @message.lines
```
Comment (paulcsmith) [inline src/exception_page/exception_page.ecr:21]: Nice 👍

**Review by paulcsmith — APPROVED** [review]
Thank you!

**Comment by paulcsmith** [comment]
Could you rebase and force push? I think it should pass with the newest stuff on master

**Comment by Sija** [comment]
@paulcsmith done, and CI is 💚

**Comment by paulcsmith** [comment]
Awesome! Thanks @Sija

**Pull request merged**